libsolidity/analysis/ViewPureChecker.cpp

```diff
@@ -94,6 +94,12 @@ class AssemblyViewPureChecker: public boost::static_visitor<void>
 		(*this)(_for.body);
 		(*this)(_for.post);
 	}
+	void operator()(yul::Break const&)
```

Comment:
Do you need to implement them here if they do nothing?

Reply:
we would have to explicitely add a default catch overload, such as: `template <typename T> void operator()(T const&) {}`;

but I'd say we stay explicit so that when we (again) add a new AST node type, we don't accidentally forget to adjust some code?

Reply:
Ah sorry, I thought it would inherit from some AST walker. In that case you of course have to implement it like that.